app/test_main.py

```diff
@@ -1,3 +1,41 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_zeros_when_inputs_are_zeros() -> None:
+    expected_value = [0, 0]
+    assert get_human_age(0, 0) == expected_value
+
+
+def test_should_return_zeros_when_values_almost_15() -> None:
+    expected_value = [0, 0]
+    assert get_human_age(14, 14) == expected_value
+
+
+def test_should_return_ones_when_values_are_exactly_15() -> None:
+    expected_value = [1, 1]
+    assert get_human_age(15, 15) == expected_value
```

Comment:
Instead of creating almost same tests you just can modify the parameters of test using `@pytest.mark.parametrize` . Aslo add negative tests cases , where you will expect some error for example : )